src/App.test.jsx

```diff
@@ -2,36 +2,18 @@
 // 🚨🚨 https://mswjs.io/docs/ 🚨🚨
 
 import { screen, render } from '@testing-library/react'
-// 🚨
-// import rest
-// import setupServer
-import App from './App'
-
-const user = {
-  id: 1,
-  created_at: '2021-12-13T00:17:29+00:00',
-  // 🚨 Add a name here
-  name: '',
-  avatar: 'https://thumbs.gfycat.com/NiceRequiredGrunion-size_restricted.gif',
-  header: 'https://static.wikia.nocookie.net/naruto/images/5/50/Team_Kakashi.png',
-  likes: ['React', 'Anime', 'Traveling', 'Living', 'Tower Defense Games', 'Card Games'],
-  motto: 'Res Non Verba',
-  color: 'crimson',
-}
 
-// 🚨 Create your server
+import App from './App'
 
-// 🚨 Listen for server start
-beforeAll()
 
-// 🚨 Close server when complete
-afterAll()
 
```

Comment:
you will need to import your `{ server } `in this file and invoke it with `server.use()`